src/callbacks/callbacks.jl

```diff
@@ -105,9 +105,8 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
     sfc_T = Fields.array2field(rrtmgp_model.surface_temperature, axes(sfc_ts))
     @. sfc_T = TD.air_temperature(thermo_params, sfc_ts)
 
-    ᶜp = Fields.array2field(rrtmgp_model.center_pressure, axes(Y.c))
+    rrtmgp_model.center_pressure .= Fields.field2array(ᶜp)
```

Comment:
Was there an issue with the pressure or is this just an optimization?

Reply:
There was no issue, but I think it's better not to calculate it again if we already have it in precomputed quantities.